pkg/asset/agent/image/agentartifacts.go

```diff
@@ -73,6 +75,8 @@ func (a *AgentArtifacts) Generate(_ context.Context, dependencies asset.Parents)
 
 	if agentconfig.Config != nil {
 		a.BootArtifactsBaseURL = strings.Trim(agentconfig.Config.BootArtifactsBaseURL, "/")
+		// External platform will always create a minimal ISO
+		a.MinimalISO = agentconfig.Config.MinimalISO || agentManifests.AgentClusterInstall.Spec.PlatformType == hiveext.ExternalPlatformType
```

Comment:
This could be somehow confusing, especially if the user accidentally (or not) set to true the `minimalISO` flag. In case of `External` platform type I'd found clearer to have a log trace informing the user that the minimal ISO would be generated due the platform type (or something similar)

Reply:
Added info messages to show reason for creating minimal ISO.